src/components/AddForm/AddForm.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react';
+import { Users } from '../../types/Users';
+
+type Props = {
+  users: Users[];
+};
+
+export const AddForm: React.FC<Props> = props => {
+  const { users } = props;
+
+  const [title, setTitle] = useState<string>('');
+  const [userId, setUserId] = useState<number>(0);
+
+  const onTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value);
+  };
+
+  const onUserIdChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    setUserId(+event.target.value);
+  };
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+  };
+
+  return (
+    <form action="/api/todos" method="POST" onSubmit={handleSubmit}>
+      <div className="field">
+        <label htmlFor="">
+          Name:
+          <input
+            type="text"
+            data-cy="titleInput"
+            placeholder="Enter a title"
+            value={title}
+            onChange={onTitleChange}
+          />
+        </label>
+        <span className="error">Please enter a title</span>
```

Comment:
The error message 'Please enter a title' is always displayed. Consider conditionally rendering this error message based on a state variable that tracks whether the title input is valid.